cpp/include/cudf/io/detail/avro.hpp

```diff
@@ -38,7 +39,7 @@ namespace avro {
 table_with_metadata read_avro(
   std::unique_ptr<cudf::io::datasource>&& source,
   avro_reader_options const& options,
-  rmm::cuda_stream_view stream        = rmm::cuda_stream_default,
+  rmm::cuda_stream_view stream        = cudf::default_stream_value,
```

Comment:
`rmm::cuda_stream_default` already has the meaning you want when per-thread default stream is enabled. Please stick to `rmm::cuda_stream_default`.